frontend/www/js/omegaup/login/signin.ts

```diff
@@ -27,14 +27,17 @@ OmegaUp.on('ready', () => {
     const params = new URL(document.location.toString()).searchParams;
     const pathname = params.get('redirect');
     if (pathname && pathname.indexOf('/') === 0) {
-      window.location.href = pathname;
+      const url = new URL(document.location.origin + pathname);
+      url.searchParams.set('fromLogin', '');
+      window.location.href = url.toString();
       return;
     }
+    const fromLoginParam = '?fromLogin';
     if (isAccountCreation) {
-      window.location.href = '/profile/';
+      window.location.href = '/profile/' + fromLoginParam;
```

Comment:
Acá es mejor usar [String interpolation](https://dmitripavlutin.com/string-interpolation-in-javascript/)
```suggestion
      window.location.href = `/profile/${fromLoginParam}`;
```
igual en L40